08-regression2.Rmd

```diff
@@ -56,16 +56,19 @@ its sale price? In particular, recall that we have come across a new 2,000-squar
 in purchasing with an advertised list price of
 \$350,000. Should we offer the list price, or is that over/undervalued?
 To answer this question using simple linear regression, we use the data we have
-to draw the straight line of best fit through our existing data points:
+to draw the straight line of best fit through our existing data points.
+The small subset of data as well as the line of best fit are shown
+in Figure \@ref(fig:08-lin-reg1).
 
-```{r 08-lin-reg1, message = FALSE, warning = FALSE, echo = FALSE, fig.height = 4, fig.width = 5, fig.cap = "Scatter plot of price (USD) versus house size (square footage) with line of best fit for subset of the Sacramento housing data set"}
+```{r 08-lin-reg1, message = FALSE, warning = FALSE, echo = FALSE, fig.height = 4, fig.width = 5, fig.cap = "Scatter plot of price versus house size with line of best fit for subset of the Sacramento housing data"}
 library(tidyverse)
```

Comment:
I could not execute this chapter independently without adding these two packages, I think each chapter should be able to be executed independently.
```suggestion
library(scales)
library(tidymodels)
library(tidyverse)
```

Reply:
fixed and pushed!